doc/website/blog/2023-08-16-passwordless-migrations.md

```diff
@@ -0,0 +1,162 @@
+or [single sign-on](https://en.wikipedia.org/wiki/Single_sign-on).
+
+In this post, we'll focus on the security concerns of passwords (or API Tokens) for automated systems, such as
+CI/CD pipelines, which are used to perform schema migrations. Such tokens pose a challenge to securing systems in a few ways:
+
+* **Leaks.** Passwords, when stored in configuration files, are typically in plain text, increasing the risk of leaks.
+* **Granularity.** When passwords are shared among multiple users, it becomes challenging to audit system usage and to revoke access for
+  individual users based on role changes or emerging security concerns.
+* **Visibility.** Because passwords are usually visible to operators and are shared by multiple users, it's hard to track
+  which user performed which operation once authenticated.
+* **Rotation.** Because passwords tend to be long-lived, their rotation becomes a cumbersome task.
+
+## IAM Authentication
+
+IAM, short for [Identity and Access Management](https://en.wikipedia.org/wiki/Identity_management), is a framework that
+was adopted by virtually all cloud providers for managing digital identities and their permissions. Unlike traditional 
+password-based systems where credentials are stored and checked, IAM verifies who (or what) is making a request and 
+then checks the permissions associated with that identity.
+
+IAM services supply mechanisms for generating short-lived tokens based on the identity of the caller. In addition, 
+these services provide a centralized way to manage permissions (by creating granular access policies and grouping them
+into roles) and auditing capabilities to track how subjects (users or services) use the system.
+
+Configured correctly, under IAM, every subject can access exactly what it needs, and nothing more, without
+ever using a password or other token that can be leaked or stolen. When a person leaves your organization, or no 
+longer needs access to a particular resource, you can revoke their access by updating their IAM role.
+
+## IAM authentication for Databases
+
+Most databases that are in use today predate IAM and have developed their own internal mechanisms for authentication
+and authorization. In recent years, cloud vendors have worked to create a bridge between IAM and databases, allowing
+users to authenticate to databases using IAM credentials.  In this post, we'll focus on AWS's implementation of IAM
+authentication for RDS databases.
+
+#### How does it work? 
+First, you enable IAM authentication on your RDS instance. Essentially this installs a plugin on the database that
+allows it to authenticate users using IAM credentials instead of passwords. Read how to do this in the 
+[AWS documentation](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.IAMDBAuth.Enabling.html)
+
+Next, you create a database user and grant it permission to authenticate using IAM. 
+  
+In MySQL, you execute a statement like this:
```

Comment:
@rotemtam Tabs container for the MySQL and PostgreSQL options: https://github.com/ariga/atlas/compare/master...gedalyah-ariga-patch-1?quick_pull=1

Reply:
```
import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';
import InstallationInstructions from '../components/_iam_auth_blog.mdx'
```

Reply:
components are relevant for places we want to reuse the same snippet. this isn't such a case